articles/enroll-personal-byod-ios-ipad-hosts-with-managed-apple-account.md

```diff
@@ -0,0 +1,75 @@
+# Enroll personal (BYOD) iPhones and iPads with Managed Apple Account
+
+![Apple Account-driven User Enrollment (BYOD)](../website/assets/images/articles/apple-account-driven-user-enrollment.png)
+
+_Available in Fleet Premium._
+
+In Fleet, you can allow your end users to enroll their personal iPhones and iPads to Fleet using [Account-driven User Enrollment](https://support.apple.com/en-gb/guide/deployment/dep23db2037d/web).
+
+With Account-driven User Enrollment, end users can separate work and personal data using their [Managed Apple Account](https://support.apple.com/en-gb/guide/apple-business-manager/axm78b477c81/web). End users retain privacy over their personal information, while IT admins manage work-related OS settings and applications.
+
+- [Step 1: Connect Apple Business Manager (ABM) to Fleet](#step-1-connect-apple-business-manager-abm-to-fleet)
+- [Step 2: Add and verify your domain in Apple Business Manager (ABM)](#step-2-add-and-verify-your-domain-in-apple-business-manager-abm)
+- [Step 3: Connect (federate) your identity provider (IdP) with Apple Business Manager (ABM)](#step-3-connect-federate-your-identity-provider-idp-with-apple-business-manager-abm)
+- [Step 4: Create a `📱🔐 Personal mobile devices` team](#step-4-create-personal-mobile-devices-team)
```

Comment:
```suggestion
- [Step 4: Create a `📱🔐 Personal mobile devices` team](#step-4-create-a-code-personal-mobile-devices-code-team)
```

Looks like the fleetdm.com website expects "code" in the anchor link when there are backticks (`) in the section header.

Reminder to test guides locally! https://fleetdm.com/handbook/it-and-enablement#test-fleetdm-com-locally